### PR TITLE
Dev fetch money

### PR DESCRIPTION
**Pull request by Jenjamin3000**
In this PR:

- The useless NavigationDrawerActivity has been removed
- There is money! This is fetched from the database and we can spend it on power ups

Note: For now, we can just participate to the fund but not really buy power ups! This is for the backend.

### CODE REVIEW

File: app/src/main/java/com/github/campus_capture/bootcamp/firebase/BackendInterface.java
Comment (Mopplikus) [inline app/src/main/java/com/github/campus_capture/bootcamp/firebase/BackendInterface.java:1]: Good, although using no futures might pose a problem

File: app/src/main/java/com/github/campus_capture/bootcamp/firebase/FirebaseBackend.java
Comment (Mopplikus) [inline app/src/main/java/com/github/campus_capture/bootcamp/firebase/FirebaseBackend.java:1]: Good! Once again, same issue. Imagine the connection is broken, the user loses money in the UI and yet the server knows nothing about it. Again, futures will solve this here

**Review by Mopplikus — APPROVED** [review]
Good changes! As I've mentioned, we'll still be needing the futures to make sure that the operations are safe and users don't lose or gain funds through some bug, but don't worry, I'll solve these issues soon enough

File: app/src/main/java/com/github/campus_capture/bootcamp/firebase/FirebaseBackend.java
Comment (Jenjamin3000) [reply app/src/main/java/com/github/campus_capture/bootcamp/firebase/FirebaseBackend.java:1]: Don't worry! The real money is fetched from the DB. The "local money" is just to have the impression to really have spend something when we click on the button.

**Pull request merged**